Source/Core/DolphinQt/Config/Graphics/AdvancedWidget.cpp

```diff
@@ -185,13 +185,23 @@ void AdvancedWidget::CreateWidgets()
   dump_layout->addWidget(m_frame_dumps_resolution_type, 0, 1);
 
 #if defined(HAVE_FFMPEG)
-  m_dump_use_ffv1 =
-      new ConfigBool(tr("Use Lossless Codec (FFV1)"), Config::GFX_USE_FFV1, m_game_layer);
+  // Update EnableBitrate() if options are changed.
+
+  const std::vector<std::pair<QString, QString>> options{
+      {tr("Auto"), QStringLiteral("")},
+      {QStringLiteral("MPEG4"), QStringLiteral("mpeg4")},
+      {QStringLiteral("FFV1"), QStringLiteral("ffv1")},
+      {QStringLiteral("UTVideo"), QStringLiteral("utvideo")}};
```

Comment:
Legend (from mediainfo:

```
Format                      : MPEG-4 Visual
Format settings, Matrix     : Default (H.263)
Codec ID                    : XVID
Codec ID/Hint               : XviD
```

```
Format                      : RGB
Codec ID                    : ULRG
Codec ID/Info               : Ut Video Lossless Codec
Codec ID/Hint               : Ut Video
```

```
Format                      : FFV1
Codec ID                    : FFV1
```

I think it's better to name the options after codec names, so `MPEG4` (which means format) would become `XviD`, and `UTVideo` - `Ut Video`.